_data/previous_presentations.yaml

```diff
@@ -1,4 +1,13 @@
-- title: Hydra in a Box
+- title: Building and Bundling a National Digital Platform
+  presenters:
+    - Mark Matienzo
+    - Debra Hanken Kurtz
+    - Tom Cramer
+  event: CNI Fall 2015 Membership Meeting
+  location: "Washington, DC"
+  date: 2015-12-15
+  url:  https://osf.io/h4ru8/
+  - title: Hydra in a Box
```

Comment:
Methinks the two spaces here (before "- title") are a typo, and they break site generation:

```
$ jekyll s
Configuration file: /home/mjg/workspace/hybox.github.io/_config.yml
            Source: /home/mjg/workspace/hybox.github.io
       Destination: /home/mjg/workspace/hybox.github.io/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
jekyll 3.0.0 | Error:  (/home/mjg/workspace/hybox.github.io/_data/previous_presentations.yaml): did not find expected key while parsing a block mapping at line 1 column 3
```